source/unified-test-format/unified-test-format.rst

```diff
@@ -1779,6 +1779,54 @@ The following arguments are supported:
 
 - ``session``: Optional string. See `commonOptions_session`_.
 
+runCursorCommand
+~~~~~~~~~~~~~~~~
+
+`Generic cursor returning command runner <../run-command/run-command.rst>.`__
+
+This method does not inherit a read concern or write concern (per the
+`Read and Write Concern <../read-write-concern/read-write-concern.rst#generic-command-method>`__
+spec), nor does it inherit a read preference (per the
+`Server Selection <../server-selection/server-selection.rst#use-of-read-preferences-with-commands>`__
+spec); however, they may be specified as arguments.
+
+This operation proxies the databases's ``runCursorCommand`` method and supports the same arguments and options.
+
+When executing the provided command, the test runner MUST fully iterate the cursor.
+This will ensure consistent behavior between drivers that eagerly create a server-side cursor and those that do so lazily when iteration begins.
+
+The following arguments are supported:
+
+- ``command``: Required document. The command to be executed.
+
+- ``commandName``: Required string. The name of the command to run. This is used
+  by languages that are unable preserve the order of keys in the ``command``
+  argument when parsing YAML/JSON.
+
+- ``readPreference``: Optional object. See `commonOptions_readPreference`_.
+
+- ``session``: Optional string. See `commonOptions_session`_.
+
+- ``batchSize``: Optional int32. Use this value to configure the batchSize sent on the getMore command.
+
+- ``maxTimeMS``: Optional int32. Use this value to configure the maxTimeMS sent on the getMore command.
+
+- ``comment``: Optional BSON value. Use this value to configure the comment sent on the getMore command.
+
+- ``cursorType``: Optional string enum value, one of ``'tailable' | 'tailableAwait' | 'nonTailable'``. Use this value to configure the enum passed to the ``cursorType`` option.
+
+- ``timeoutMode``: Optional string enum value, one of ``'iteration' | 'cursorLifetime'``. Use this value to configure the enum passed to the ``timeoutMode`` option.
+
+createRunCursorCommand
+~~~~~~~~~~~~~~~~~~~~~~
+
+This operation proxies the database's ``runCursorCommand`` method and supports the same arguments and options.
+Test runners MUST ensure that the server-side cursor is created (i.e. the command document has executed) as part of this operation and before the resulting cursor might be saved with `operation.saveResultAsEntity <operation_saveResultAsEntity_>`_.
+Test runners for drivers that lazily execute the ``runCursorCommand`` command on the first iteration of the cursor MUST iterate the resulting cursor once.
+The result from this iteration MUST be used as the result for the first iteration operation on the cursor.
+
+Test runners MUST NOT iterate the resulting cursor when executing this operation and test files MUST NOT specify `operation.expectResult <operation_expectResult_>`_ for this operation.
```

Comment:
```suggestion
Test runners MUST NOT iterate the resulting cursor when executing this operation and test files SHOULD NOT specify `operation.expectResult <operation_expectResult_>`_ for this operation.
```

"SHOULD NOT" is used instead of "MUST NOT" since we cannot enforce this with the schema. See: [UTF: Design Rationale](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#design-rationale).

It's for that reason the original language for `createFindCursor` uses "SHOULD NOT" and we _do not_ have tests for the behavior.

Reply:
updated, ty for the design context